RPGManager_development/src/rpg_database/character_sheet/Languages.java

```diff
@@ -0,0 +1,26 @@
+package rpg_database.character_sheet;
+
+import java.security.InvalidParameterException;
+import java.util.AbstractCollection;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import rpg_database.character_sheet.interfaces.CustomSetter;
+import rpg_database.character_sheet.interfaces.MultipleFieldsGetterSetter;
+
+public enum Languages {
+	ANDER("Ander"), ANTIVAN("Antivan"), CIRIANE("Ciriane"), TRADE_TONGUE("Trade tongue"), DWARVEN_LANGUAGES("Dwarven languages"),
+	ELVISH("Elvish"), QUNLAT("Qunlat"), ORLESIAN("Orlesian"), RIVAINI("Rivaini"), TEVENE("Tevene"), ANCIENT_TEVENE("Ancient Tevene");

```

Comment:
Ciriane, the language of the Ciriane tribe who first settled central Orlais. 
---> http://dragonage.wikia.com/wiki/Language <---
De töröltem, mert nincs olyan background ami használná. 

Módosítottam a többit ahogy kérted.